src/core/XRPValue.js

```diff
@@ -0,0 +1,59 @@
+/* @flow */
+
+'use strict';
+
+const GlobalBigNumber = require('bignumber.js');
+const BigNumber = GlobalBigNumber.another({
+  ROUNDING_MODE: GlobalBigNumber.ROUND_HALF_UP,
+  DECIMAL_PLACES: 40
+});
+
+const Value = require('./value').Value;
+
+class XRPValue extends Value {
+
+  constructor(value: string | BigNumber) {
+    super(value);
+    this.rippleUnits = new BigNumber(1e6);
```

Comment:
This can be a global constant in this module and IOUValue.js, which would actually reduce it's exposure. Also I just noticed, we should probably stick to the file naming convention of all lowercase, no separator between words.